core/helpers/EEH_Line_Item.helper.php

```diff
@@ -1620,12 +1891,16 @@ public static function non_empty_line_items(EE_Line_Item $line_item)
      *
      * @param EE_Line_Item $line_item
      * @return EE_Line_Item
-     * @throws \EE_Error
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     * @throws ReflectionException
      */
     public static function non_empty_line_item(EE_Line_Item $line_item)
     {
         if ($line_item->type() === EEM_Line_Item::type_line_item &&
-            $line_item->OBJ_type() === 'Ticket' &&
+            $line_item->OBJ_type() === EEM_Line_Item::OBJ_TYPE_TICKET &&
```

Comment:
heh, javascript standards for multiline conditionals have crept in here :)  For php the `&&` are supposed to go at the beginning of the line right?

Reply:
DOH!!! 🤦‍♂️